tasks/www.rb

```diff
@@ -44,6 +44,7 @@
   task :assemble do
     Log.section 'Copy only tutorial into middleman build directory'
     sh('rsync -a --exclude=index.html www/tutorial/dist/* www/build/')
+    sh('cp www/tutorial/dist/index.html www/build/tutorial.html')
```

Comment:
Yes :) see line 46 there, it will sync all the others, excluding `index.html` because it exists already. This MR just adds back the `index.html` and renames it to `tutorial.html`